content/sensu-core/1.4/guides/monitor-the-monitor.md

```diff
@@ -0,0 +1,197 @@
+---
+title: "Monitoring Sensu with Sensu"
+description: "Strategies and best practices for monitoring Sensu with Sensu"
+version: "1.4"
+weight: 10
+previous: ../securing-redis
+menu:
+  sensu-core-1.4:
+    parent: guides
+---
+
+In this guide, we'll walk you through the best practices and strategies for monitoring Sensu with Sensu. By the end of the guide, you should have a thorough understanding of what is required to ensure your Sensu components are properly monitored, including:
+
+* [How to monitor your Sensu Server instance(s)](#monitoring-sensu-server)
+* [How to monitor your Sensu API instance(s)](#monitoring-sensu-api)
+* [How to monitor your Uchiwa Dashboard instance(s)](#monitoring-uchiwa-dashboard)
+* [How to monitor your RabbitMQ instance(s)](#monitoring-rabbitmq)
+* [How to monitor your Redis instance(s)](#monitoring-redis)
+
+In order to completely monitor a Sensu stack (Sensu services, Redis, RabbitMQ), you will need to have at least one other independent Sensu stack to do so. This is due to a single Sensu stack can not monitor itself completely, as if some components are down, Sensu will not be able to alert on itself properly
+
+_NOTE: This guide assumes you are not using Sensu clustering, RabbitMQ clustering or using Redis Sentinels. You can still monitor each server using the below described strategies, but it may not be the best way to do so._
```

Comment:
Instead of "but it may not be the best way to do so," maybe "but note that in order to effectively monitor clustered instances, you'll need to employ a different methodology."